src/components/Tokens/TokenTable/TokenRow.tsx

```diff
@@ -304,6 +304,11 @@ export const HEADER_DESCRIPTIONS: Record<TokenSortMethod, ReactNode | undefined>
       Total value locked (TVL) is the aggregate amount of the asset available across all Uniswap v3 liquidity pools.
     </Trans>
   ),
+  [TokenSortMethod.FULLY_DILUTED_VALUATION]: (
+    <Trans>
+      Fully diluted valuation is the market capitalization of an asset if maximum token supply were in circulation.
```

Comment:
can we also add `(FDV)` after `Fully diluted valuation` similar to what we have for `TVL` above?